challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/components/Clock.vue

```diff
@@ -0,0 +1,102 @@
+<script setup>
+import { ref} from "vue";
+
+const timeRange = ref({
+  start: Number,
+  end:Number
+});
+
+const props = defineProps(['activeHours'])
+const checkValid = () => {
+  timeRange.value = props.activeHours;
+  if( timeRange.value.start < 0  )  timeRange.value.start = 0;
+  if( timeRange.value.start > 24)  timeRange.value.end = 24;
+};
+checkValid();
+
+</script>
+
+<template>
+
+    <div class="timeRange"
+         v-for="(activeHour,index) in timeRange"
+         :key="index"
+         :style="{backgroundImage : 'conic-gradient(rgb(255 255 255 / 0%)  00deg  , rgb(255 255 255 / 0%) '+activeHour.start *15+'deg ,#ee7e25  '+activeHour.start *15+'deg  , #ee7e25  '+activeHour.end *15+'deg,rgb(255 255 255 / 0%)'+activeHour.end *15+'deg  , rgb(255 255 255 / 0%) 360deg)' }"
```

Comment:
What do first and last arguments of `conic-gradient` do? Nothing changed when I deleted them.